src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/tests/adapters/TreeTableAdapter.test.js

```diff
@@ -78,6 +78,114 @@ test('Render data with schema', () => {
     expect(treeListAdapter).toMatchSnapshot();
 });
 
+test('Render different kind of data with schema', () => {
```

Comment:
Same as above, can't we integrate that in the previous test? Also the name `different kind of data` is not really clear...